tensorflow/compiler/mlir/lite/transforms/prepare_composite_functions_tf.cc

```diff
@@ -281,6 +281,8 @@ void PrepareCompositeFunctionsPass::ConvertTFImplements(FuncOp func,
     if (failed(image_warping.RewriteFunc())) {
       return signalPassFailure();
     }
+  } else {
+      func.emitWarning() << "Function was flagged with experimental_implements, but no handler picked it up";
```

Comment:
func.emitWarning() << "with the 'xyz' attribute does not have the corresponding fused implementation found.";

Each case does have a different attribute name.

Reply:
@abattery I can change the wording, but for at least the first two cases, the attribute name is actually the same (experimental_implements), they are instead distinguished by whether the key was a plain string, or could be parsed via the attr_value.proto structure.
In terms of the last one (which internally uses the string "tf.api_implements"), I actually don't know what to use there at all, this string only appears inside tensorflow/compiler/mlir/lite/tests/prepare-composite-functions-tf.mlir, i.e. an integration test. It is not mentioned in any doc at all.

If you could let me know what explicit strings you would like to see for those 3 warnings,  I think that may be the fastest way.
Thanks!